deployment/helm/node-feature-discovery/templates/master.yaml

```diff
@@ -55,11 +56,16 @@ spec:
             name: grpc
           - containerPort: {{ .Values.master.metricsPort | default "8081" }}
             name: metrics
+          - containerPort: 8082
```

Comment:
do we need to make this one configurable for the host network case ?

i.e also add cmdline flag since currently AFAIU value is fixed [1]

[1] https://github.com/kubernetes-sigs/node-feature-discovery/blob/c0561631d62b18e1cb97334f1f32f35243262fba/cmd/nfd-worker/main.go#L36

Reply:
See PR description :)

> out-of-scope: will make the health port configurable in a follow-up PR to be able to run all daemonsets in the host's network namespace (i.e., avoid port conflicts)

Reply:
oh sorry :)